instrumentation/jdbc/library/src/main/java/io/opentelemetry/instrumentation/jdbc/internal/JdbcInstrumenterFactory.java

```diff
@@ -30,19 +34,38 @@ public static Instrumenter<DbRequest, Void> createStatementInstrumenter() {
 
   public static Instrumenter<DbRequest, Void> createStatementInstrumenter(
       OpenTelemetry openTelemetry) {
+    return createStatementInstrumenter(
+        openTelemetry,
+        true,
+        ConfigPropertiesUtil.getBoolean(
+            "otel.instrumentation.common.db-statement-sanitizer.enabled", true));
+  }
+
+  public static Instrumenter<DbRequest, Void> createStatementInstrumenter(
+      OpenTelemetry openTelemetry, boolean enabled, boolean statementSanitizationEnabled) {
     return Instrumenter.<DbRequest, Void>builder(
             openTelemetry,
             INSTRUMENTATION_NAME,
             DbClientSpanNameExtractor.create(dbAttributesGetter))
         .addAttributesExtractor(
             SqlClientAttributesExtractor.builder(dbAttributesGetter)
-                .setStatementSanitizationEnabled(
-                    ConfigPropertiesUtil.getBoolean(
-                        "otel.instrumentation.common.db-statement-sanitizer.enabled", true))
+                .setStatementSanitizationEnabled(statementSanitizationEnabled)
```

Comment:
Just double checking here. People are ok if the builder overrides what's in system properties?

Reply:
I think it is fine. Usually we don't use the configuration properties for manual instrumentation at all. I guess this is a bit special because when using the driver instrumentation the only way is to use configuration properties.